setup.py

```diff
@@ -22,8 +22,7 @@
     packages=find_packages(exclude=['*.pyc']),
     include_package_data=True,
     dependency_links=[
-        'git+git://github.com/dimagi/dimagi-utils.git@24e1eaad37ac735ab9309253011f09835c9ee67e#egg=dimagi-utils-1.3usec',
-        'git+git://github.com/dimagi/django-soil.git@ad98c51dd95baec568c5f83af0807ef7965bf201#egg=django-soil-0.10.0',
-        'git+git://github.com/smartfile/django-transfer.git@6e0dc94c3341c358fca8eb2bf74e23aee3983ec4#egg=django-transfer-dev',
```

Comment:
It was intentional as it didn't make a difference for me when I ran these tests and it seemed better to be pulling down a real version vs pointing at a random commit. I looked at hq though and it's also pinned to this version, so it probably makes a difference there. Going to pin for now